File: fuelnet_back/src/main/java/com/fuelnet/fuelnet/dto/InventoryMovementResponse.java
package com.fuelnet.fuelnet.dto;

import com.fuelnet.fuelnet.enums.FuelType;
import lombok.Builder;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
public class InventoryMovementResponse {
    private Long id;
    private FuelType fuelType;
    private BigDecimal gallonsAdded;
    private BigDecimal levelBefore;
    private BigDecimal levelAfter;
    private BigDecimal fillPercentageAfter;
    private String supplier;
    private LocalDateTime rechargeDate;
    private String notes;
    private String registeredBy;
}

File: FuelControl/app/src/main/java/co/edu/unipiloto/fuelcontrol/api/requests/InventoryMovementResponse.kt
package co.edu.unipiloto.fuelcontrol.api.requests

data class InventoryMovementResponse(
    val id: Long,
    val fuelType: String,
    val gallonsAdded: Double,
    val levelBefore: Double,
    val levelAfter: Double,
    val fillPercentageAfter: Double,
    val supplier: String?,
    val rechargeDate: String,
    val notes: String?,
    val registeredBy: String
)